inspire/base/static/js/search/search_results.js

```diff
@@ -44,8 +44,111 @@ require(['jquery', 'bootstrap'], function($) {
         $("#download" + data.recid).attr("href", "/formatter/download-latex/latex_us/" + data.recid)
       });
     });
+
   })
 
+  $("#export-as-bibtex").on('click', function() {
+    if (sList != 0) {
+      create_url('/formatter/export-as/bibtex/', sList);
+    }
+    $("#dropdown-export").html('BibTex <span class="caret"></span>');
+  });
+
+  $("#export-as-latex-eu").on('click', function() {
+    if (sList != 0) {
+      create_url('/formatter/export-as/latex_eu/', sList);
+    }
+    $("#dropdown-export").html('LaTex(EU) <span class="caret"></span>');
+  });
+
+  $("#export-as-latex-us").on('click', function() {
+    if (sList != 0) {
+      create_url('/formatter/export-as/latex_us/', sList);
+    }
+    $("#dropdown-export").html('LaTex(US) <span class="caret"></span>');
+  });
+
+  $("#export-as-cv-latex").on('click', function() {
+    if (sList != 0) {
+      create_url('/formatter/export-as/cv_latex/', sList);
+    }
+    $("#dropdown-export").html('CV format (LaTex) <span class="caret"></span>');
+  });
+
+  $("#export-as-cv-html").on('click', function() {
+    if (sList != 0) {
+      create_url('/formatter/export-as/cv_latex_html/', sList);
+    }
+    $("#dropdown-export").html('CV format (html) <span class="caret"></span>');
+  });
+
+  $("#export-as-cv-text").on('click', function() {
+    if (sList != 0) {
+      create_url('/formatter/export-as/cv_latex_text/', sList);
+    }
+    $("#dropdown-export").html('CV format (text) <span class="caret"></span>');
+  });
+
+  var sList = [];
+  $('#checkbox-select-all').click(function(event) {
+    if (this.checked) {
+      $('.checkbox-results').each(function() {
+        this.checked = true;
+        sList.push($(this).attr("id"));
+      });
+    } else {
+      $('.checkbox-results').each(function() {
+        this.checked = false;
+      });
+      sList.length = 0;
+    }
+  });
+
+  $('input[type=checkbox]').change(function() {
```

Comment:
Is this selecting all checkboxes in the page? Make sure only the ones you want get selected (search results) as there might be other checkboxes in the page that should be ignored (like the facets)